scripts/hel-gis-data-variables.yaml

```diff
@@ -0,0 +1,21 @@
+environment:
+  dev:
+    fileshare: http://edit-here/fileshare-for-dev
+  test:
+    fileshare: http://edit-here/fileshare-for-test
```

Comment:
I would suggest a slight change here:

- This file could be name `hel-gis-data-variables.yaml.example`
- The README could advice to copy the example file to the actual `hel-gis-data-variables.yaml` and edit the addresses
- `.gitignore` could include the `scripts/hel-gis-data-variables.yaml` file.

This way there's a smaller risk that the actual addresses are pushed to Github and this file doesn't show up as changed with `git status` and other git tools.

Reply:
Good points.